src/main/java/org/terasology/logic/inventory/ItemCommands.java

```diff
@@ -33,15 +35,29 @@
 import org.terasology.logic.permission.PermissionManager;
 import org.terasology.network.ClientComponent;
 import org.terasology.registry.In;
+import org.terasology.registry.Share;
+import org.terasology.utilities.Assets;
+import org.terasology.world.block.BlockManager;
+import org.terasology.world.block.BlockUri;
 import org.terasology.world.block.entity.BlockCommands;
+import org.terasology.world.block.family.BlockFamily;
+import org.terasology.world.block.items.BlockItemComponent;
+import org.terasology.world.block.loader.BlockFamilyDefinition;
+import org.terasology.world.block.shapes.BlockShape;
 
+import java.util.ArrayList;
 import java.util.Collections;
+import java.util.Iterator;
 import java.util.List;
+import java.util.Optional;
 import java.util.Set;
 
 @RegisterSystem
+@Share(ItemCommands.class)
```

Comment:
Why is this necessary? If in doubt, add a todo to track this question right here in the code.